rpcs3/stdafx.h

```diff
@@ -3,6 +3,7 @@
 #include "util/types.hpp"
 #include "util/atomic.hpp"
 #include "util/endian.hpp"
+#include "Utilities/Config.h"
```

Comment:
why?

Reply:
I think I measured build time with ClangBuildAnalyzer and it showed `Config.h` at the top.